cosmos/providers/dbt/core/operators/local.py

```diff
@@ -125,6 +125,14 @@ def run_command(
                 tmp_project_dir,
             )
 
+            # ensure there's no profiles.yml file in the project dir
+            profiles_path = os.path.join(tmp_project_dir, "profiles.yml")
+            if os.path.exists(profiles_path):
+                logger.warning(
+                    "Cosmos doesn't support user-supplied profiles.yml files. Removing..."
```

Comment:
It may be worth to mention this is a limitation of the local execution mode:
```suggestion
                    "Cosmos doesn't support user-supplied profiles.yml files when using local execution mode."
```